openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -125,27 +163,28 @@
          * Upon request, loads the staff grade/assessment section of the staff area.
          * This allows staff grading when staff assessment is a required step.
          *
-         * @param {boolean} clearAndCollapse if true, clear the staff grade form and collapse it. Otherwise
-         *     render the staff grade form if is not already loaded.
          * @returns {promise} A promise representing the successful loading
          * of the staff grade (assessment) section.
          */
-        loadStaffGradeForm: function(clearAndCollapse) {
+        loadStaffGradeForm: function() {
             var view = this;
             var $staffGradeTab = $('.openassessment__staff-grading', this.element);
+            var $staffGradeControl = $staffGradeTab.find('.staff__grade__control');
             var deferred = $.Deferred();
             var showFormError = function(errorMessage) {
                 $staffGradeTab.find('.staff__grade__form--error').text(errorMessage);
             };
 
-            if (clearAndCollapse) {
-                // Collapse the editor and update the counts.
-                $staffGradeTab.find('.staff__grade__control').toggleClass('is--collapsed', true);
-                $staffGradeTab.find('.staff__grade__form').replaceWith('<div class="staff__grade__form"></div>');
-                view.updateStaffGradeCounts();
+            $staffGradeControl.toggleClass('is--showing', true);
+            if (this.staffGradeFormLoaded) {
+                $staffGradeControl.find('.ui-slidable__content').slideDown(
+                    0,  // If transition is visible, the shrinking/growing of the form is seen.
+                    function() {  // This is to work around a JQuery bug (fixed in recent versions of JQuery).
+                        $('.ui-slidable__content').css('overflow', 'visible');
```

Comment:
My understanding was that the animation of sliding up/down needed a fixed opened height in order to animate. Since the height was variable I suggested using `$.slideUp()` and `$.slideDown()` which does basically the animation but doesn't require the fixed heights.

Not sure how important the animations are, but the above mixins would hide content visually and from screen readers, but would not have the sliding animation.

If the animations are important then I'd do something like the following:
```
$(el).slideUp().removeClass('is-expanded');
...
$(el).slideDown().addClass('is-expanded');
```
And then have all other styles - unrelated to height and display, which are handled by jQuery - part of the `is-expanded` class. This may be padding, margins, font styling, etc.